### ISSUE TITLE
Improved time markers

### ISSUE THREAD

**Issue by michael** [substance/archivist#56]
Time markers implementation in editor is not good at the moment:

- hard to realize that cursor in paragraph triggers the timemarker input field to update
- when scrolling through the doc, there is no visual clue about where is a time mark
- editing is tedious because you need to leave the text editing flow to use the input field in the toolbar

This is what we actually want for the editor:

![image](https://cloud.githubusercontent.com/assets/2931/5854952/48a58508-a233-11e4-8673-cacdab9e0adb.png)

- a special inline element for timemarks (u can use regular cursor to edit it)
- it could provide assisted editing so you don't have to type the dots for instance
- those time marks can be put anywhere, not just at the beginning of the paragraph
- in the interview reader we don't show them like this, we can use any other visual encoding but for the editor it's nice to have them available in the flow of editing

What next: I would like to disable the current timemark editing functionality. Instead editors should just leave the timemarks as they are currently in the wordfile `{00.00.13}`. We will implement that feature as proposed in the coming weeks and then we write a little converter that parses the doc for {xx.xx.xx} occurences and turns them into those inline elements we need.

@Integral you fine with that approach?

**Comment by Integral** [comment]
Definitely!

**Comment by michael** [comment]
OK. then i disable current timemark editor so people don't get confused.

**Comment by michael** [comment]
Just note for the future when we implement this: 

Idea for conversion of {00.00.00} could be that we implement a macro that recognizes that structure and turns it into the corresponding timemark element. That way we would not need to implement a server-side migration script.. instead.. once this feature is in place, the next time you open a document the macro will be applied and all {xx.xx.xx} occurences will be converted. Yay \o/
